src/__test__/country.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { HashRouter as Router } from 'react-router-dom';
+import store from '../Redux/configureStore';
+import Info from '../components/country/country';
+
+describe('info Component test', () => {
+  it('Renders Info link', () => {
+    const app = renderer
+      .create(
+        <Provider store={store}>
+          <Router>
+            <Info />
+          </Router>
+        </Provider>,
+      )
+      .toJSON();
+    expect(app).toMatchSnapshot();
+  });
```

Comment:
- [ ] please make sure that the test are passing right now they failed in my machine but in case they pass in your machine please try to provide a screenshot in the pull request
![image](https://user-images.githubusercontent.com/61790587/175402785-a2c1b711-e356-4a4f-aa02-7de774164c6a.png)